grants-stack/packages/grant-explorer/src/features/api/voting.ts

```diff
@@ -283,6 +293,46 @@ export const prepareAllocationData = ({
   return data;
 };
 
+const abi = parseAbi([
+  "function getPool(uint256) view returns ((bytes32 profileId, address strategy, address token, (uint256,string) metadata, bytes32 managerRole, bytes32 adminRole))",
+  "function usedRoundNullifiers(address, uint256) view returns (bool)",
+]);
+const alloContractAddress =
+  "0x1133ea7af70876e64665ecd07c0a0476d09465a1" as `0x${string}`;
+
+const ZuPassRegistryAddress = getAddress(
+  "0x455cC27badb067cb9b7cdE52F153DfebC83B1A99"
+);
```

Comment:
we should reate this more dynamic. different chains can have different addresses. maybe you can use a switch case and return the addresses based on the network.

Also it might makes sense to put this into common